.bazelrc

```diff
@@ -9,3 +9,12 @@ test:ci --test_strategy=standalone
 
 common --lockfile_mode=update
 test --test_output=errors
+
+build:incompatible --incompatible_load_proto_rules_from_bzl
+build:incompatible --incompatible_enable_cc_toolchain_resolution
```

Comment:
This is already flipped in Bazel 7 and will be removed

Reply:
Removed